azure-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+# PR + Master build
+name: PR Build for New Relic OpenTelemetry Java Exporters
```

Comment:
Name needs to be updated for this project?

Reply:
details details.  ;)